testbed/integration/config_examples_test.go

```diff
@@ -0,0 +1,110 @@
+package integration
```

Comment:
@evan-bradley do you think this package name makes sense? I thought about `samples_e2e` as well, but not sure.

Reply:
I think this is fine (we're checking that our configs and Collector are compatible), but I don't think the package name is paramount; `samples_e2e` would be fine too. We can always change it later.